editor/script_create_dialog.cpp

```diff
@@ -884,30 +884,49 @@ ScriptLanguage::ScriptTemplate ScriptCreateDialog::_parse_template(const ScriptL
 			String line = file->get_line();
 			if (line.begins_with(meta_prefix)) {
 				// Store meta information
-				line = line.substr(meta_prefix.length(), -1);
-				if (line.begins_with("name")) {
-					script_template.name = line.substr(5, -1).strip_edges();
+				line = line.substr(meta_prefix.length());
+				if (line.begins_with("name:")) {
+					script_template.name = line.substr(5).strip_edges();
 				}
-				if (line.begins_with("description")) {
-					script_template.description = line.substr(12, -1).strip_edges();
+				if (line.begins_with("description:")) {
+					script_template.description = line.substr(12).strip_edges();
 				}
-				if (line.begins_with("space-indent")) {
-					String indent_value = line.substr(17, -1).strip_edges();
+				if (line.begins_with("space-indent:")) {
+					String indent_value = line.substr(13).strip_edges();
 					if (indent_value.is_valid_int()) {
 						int indent_size = indent_value.to_int();
 						if (indent_size >= 0) {
```

Comment:
```suggestion
						if (indent_size > 0) {
```

Reply:
Is there any reason to change the original support for not converting spaces to indentation?

Reply:
I'm not sure about this, just suggested as it looks like a mistake.

Reply:
If it is a mistake it should have an error, but I think it's intentional to simply prevent spaces to be used as indentation

Reply:
That's true, will see, thank you for your feedback!

Will add an error though as it doesn't error out if you have a negative number